src/documents/signals/handlers.py

```diff
@@ -57,7 +57,7 @@ def set_tags(sender, document=None, logging_group=None, **kwargs):
     document.tags.add(*relevant_tags)
 
 
-def run_external_script(sender, document, **kwargs):
+def run_post_consume_external_script(sender, document, **kwargs):
```

Comment:
Maybe rename it to `run_post_consume_script` such that it is in line with the environment variable (and there are no internal scripts to be run anyway).